.github/workflows/publish.yml

```diff
@@ -1,4 +1,4 @@
-name: publish
+name: Release
```

Comment:
we should rename the file as well

Reply:
@IgorGov done with https://github.com/up9inc/mizu/commit/ca55ba976de296c01e35dccd21e01c231de83dd1